lib/ExpandablePicker/ExpandablePicker.js

```diff
@@ -254,7 +254,7 @@ module.exports = kind(
 	*/
 	rendered: function () {
 		ExpandableListItem.prototype.rendered.apply(this, arguments);
-		if (!this.$.drawer.renderOnShow) this.isDrawerRendered = true;
+		if (this.getOpen()) this.isDrawerRendered = true;
```

Comment:
I think below code seems more safe to guarantee functionality of ENYO-474
{{ if (!this.$.drawer.renderOnShow || this.getOpen()) ... }}
What do you think about it?